pages.pt-BR/common/axel.md

```diff
@@ -0,0 +1,25 @@
+# axel
+
+> Acelerador de downloads.
+> Suporta HTTP, HTTPS, e FTP.
+> Página oficial: <https://github.com/axel-download-accelerator/axel>.
+
+- Fazer download de uma URL para um arquivo:
+
+`axel {{url}}`
+
+- Fazer download especificando o nome do arquivo de destino:
+
+`axel {{url}} -o {{nome_do_arquivo}}`
+
+- Fazer download usando múltiplas conexões:
+
+`axel -n {{numero_de_conexoes}} {{url}}`
```

Comment:
I believe we had agreed to allow special characters in tokens, since these are meant to be replaced anyway.

```suggestion
`axel -n {{número_de_conexões}} {{url}}`
```

Reply:
That looks weird to me but if it was previously agreed I'll go with it haha